### DISCUSSION TITLE
windows build error

### DISCUSSION THREAD

**Discussion by shikhars371** [gulpjs/gulp#2513] [Help]
hii,

I am using gulp :
CLI version: 2.3.0
Local version: 4.0.2

when i run the same code over MAC or ubuntu , rebuiling of application is fine, but on windows when im doing npm run start on my electron app, this error comes up, if this was the problem with gulp syntax support then the same error should have popped up on MAC or linux but it comes only on windows please help me, its the final satge of my app and im exhausted from past 4 days,

error:
[10:55:52] The following tasks did not complete: start,
[10:55:52] Did you forget to signal async completion?
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! dbrowser@ start: gulp start
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the dbrowser@ start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR! C:\Users\XLR8\AppData\Roaming\npm-cache_logs\2020-11-10T05_25_52_706Z-debug.log

**Comment by ghyoo91** [comment]
Any update on this discussion?
I met the same situation with this issue.
Even if the gulpfile.js is like below it fails on windows.

function dd(done) {
    done();
}

exports.default = dd;
